examples/sample_runtime/src/main.rs

```diff
@@ -273,12 +275,15 @@ fn main() {
     println!("Creating blockchain with additional runtime...");
     // Create a blockchain with the Rust runtime and our additional runtime.
     let blockchain_base = Blockchain::new(db, service_keypair.clone(), api_sender.clone());
-    let blockchain = BlockchainBuilder::new(blockchain_base, genesis)
-        .with_rust_runtime(
-            channel.endpoints.0.clone(),
-            vec![InstanceCollection::from(Supervisor)],
-        )
-        .with_additional_runtime(SampleRuntime::default())
+    let (factory, cfg) = InstanceCollection::from(Supervisor).into();
```

Comment:
Not a fan of this transform. IMO, something like this would be more readable:

```rust
let rust_runtime =
    RustRuntime::new(channel.endpoints.0.clone()).with_factory(Supervisor);
let genesis_config = GenesisConfigBuilder::with_consensus_config(genesis)
    .with_service(Supervisor::default_instance())
    .build();
```